databricks/koalas/metadata.py

```diff
@@ -31,7 +37,8 @@ class Metadata(object):
                        and the index name to be seen in Koalas DataFrame.
     """
 
-    def __init__(self, column_fields, index_info=None):
+    def __init__(self, column_fields: List[str],
+                 index_info: Optional[List[IndexInfo]] = None) -> None:
```

Comment:
can you also add type info to the internal fields below (`_column_fields`, ...)? Thanks!